test/helpers/builder/action-registry.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const { concat, merge, isFunction, map } = require('lodash/fp');
+const modelsUtils = require('../models');
+
+const stringifyDates = object =>
+  JSON.parse(
+    JSON.stringify(object, (key, value) => {
+      if (this[key] instanceof Date) {
+        return this[key].toUTCString();
+      }
+      return value;
+    })
+  );
+
+const formatFixtures = map(stringifyDates);
+
+module.exports = {
+  ct: {
+    create: contentType => {
+      let createdModel;
+
+      return {
+        async build(state) {
+          createdModel = await modelsUtils.createContentType(contentType);
+          return { ...state, models: [...state.models, createdModel] };
```

Comment:
not 100% sure but feels like we would benefit from having something like that:

```
build(ctx) {
  // do something
  
  ctx.setState(state => ({ models: [...state.models, createdModel] }))
}
```

a build method could perform some side effetcs without changing the state instead of having to return an empty object for ex.
We could also pass this to the cleanup method so it could update the state